source/funkin/editors/character/UpdateAnimScreen.hx

```diff
@@ -124,7 +124,7 @@ class UpdateAnimScreen extends UISubstateWindow
 		{
 			char.animation.addByPrefix(name, anim, fps, looped);
 			ghostChar.animation.addByPrefix(name, anim, fps, looped);
```

Comment:
You dont need the add anim lines, since that is handled by XMLUtil.addAnimToSprite

Reply:
Alright